### ISSUE TITLE
How to store cache even if some step fails?

### ISSUE THREAD

**Issue by hissssst** [nix-community/cache-nix-action#8]
Hi, thanks for this awesome action!

The question is how do I store the nix store even if some step is failing in the CI?

**Comment by deemp** [comment]
You may allow a step to fail using [continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error).

**Comment by hissssst** [comment]
@deemp the pipeline proceeds after failing, but the cache is not restored. Please reopen

**Comment by deemp** [comment]
Can you use the latests version of `cache-nix-action` and provide just its logs?

**Comment by deemp** [comment]
@hissssst, there's now the `save-always` input (see [Inputs](https://github.com/nix-community/cache-nix-action?tab=readme-ov-file#inputs)).

**Comment by hissssst** [comment]
`save-always: true` is not working

**Comment by deemp** [comment]
@hissssst, yes. This is tracked in https://github.com/nix-community/cache-nix-action/issues/36 with more details

**Comment by deemp** [comment]
@hissssst, as it was suggested in https://github.com/actions/cache/issues/1315#issuecomment-2163822166, you can use separate `restore` and `save` steps.
